### ISSUE TITLE
Hide action menu per default, use arrows to show

### ISSUE THREAD

**Issue by ffMathy** [ffMathy/Shapeshifter#258]
I thought about what you said @thegabriestein about hiding the action menu per default, and then showing it when pressing the arrow left key, as well as having an arrow icon that reveals that it is possible to do other actions.

This makes a lot of sense now that #224 is closed (because now that the UI is right-aligned it all looks a bit weird), and I will implement it as part of this issue. Prioritize it as you wish!

**Comment by thegabriestein** [comment]
Yeah as mentioned in #224, the overlay should probably stick to the right side of the screen, which means the action menu should extend out from the left side of the overlay. 

This will definitely help users with 13" laptop screens or vertical monitors etc to save a lot more important screen real estate when needed.

Probably using the Left Arrow to trigger the menu (would make the most sense), and then Right Arrow closes the action menu when it's open and switches back to navigating the item list, and pressing Right Arrow again closes the overlay (when the menu is hidden). 

So Control and V trigger the overlay; then Left Arrow expands, Right Arrow exits, Up and Down Arrows are for selecting items and actions.

Then it can simply be mirrored for the left side of the screen.
